docs/source/markdown/podman-secret-create.1.md

```diff
@@ -89,6 +89,16 @@ Create the specified secret via stdin.
 $ printf <secret> | podman secret create my_secret -
 ```
 
+Create or rotate a cryptographically secure random secret just under the maximum `512000` bytes via stdin.
+```
+openssl rand -base64 378000 | podman secret create --replace my_secret -
+```
+
+Mount local file based secret securely in a container.
```

Comment:
A nit of a nit, but since it is user facing
```suggestion
Mount a local file-based secret securely in a container.
```

Reply:
I did have it like that first - will also fix the other "local file" instances

Reply:
force pushed ;o)